soc/riscv/nordic_nrf/common/vpr/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+choice VPR_HW_STACKING_VARIANT
+	prompt "VPR hardware register stacking variant"
+	default VPR_HW_STACKING_32_BIT
+
+config VPR_HW_STACKING_32_BIT
+	bool "32 bit variant"
+	help
+	  The VPR hw stacking with 32 bit chunks
+
+config VPR_HW_STACKING_64_BIT
+	bool "64 bit variant"
+	help
+	  The VPR hw stacking with 64 bit chunks
+
```

Comment:
This is a HW property, it should be in DT.

Reply:
done